components/automate-ui/src/app/modules/policy/details/policy-details.component.html

```diff
@@ -65,9 +65,9 @@
                 {{ member.displayType }}
               </chef-table-cell>
               <chef-table-cell class="three-dot-column">
-                <chef-control-menu>
-                  <chef-option (click)="removeMember(member)">Remove Member</chef-option>
-                </chef-control-menu>
+                <mat-select panelClass="chef-control-menu-x">
+                  <mat-option (onSelectionChange)="removeMember($event, member)">Remove Member</mat-option>
```

Comment:
mat-select requires the use of `onSelectionChange` in order to be keyboard accessible.  Unfortunately this will cause a double fire of connected function -> one for the selection and one for the deselection.  The current workaround is running an if check for whether on not this was user input using $event.isUserInput.  see: https://github.com/angular/components/issues/4094#issuecomment-363746964